src/Main.java

```diff
@@ -1,5 +1,75 @@
+import java.util.Arrays;
+
+import static java.lang.System.*;
+import static java.util.Arrays.*;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
-    }
-}
\ No newline at end of file
+        System.out.println("Задание 1");
+        int os = 0;
+        if (os == 0) {
+            System.out.println("Установите версию приложения для iOS по ссылке");
+        } else if (os == 1) {
+            System.out.println("Установите версию приложения для Android по ссылке");
+        }
+        System.out.println("Задание 2");
+        int years = 2013;
+        if (os == 0);
+        if (years < 2015) {
+            System.out.println("Установите облегченную версию приложения для iOS по ссылке");
+        } else {
+            System.out.println("Установите версию приложения для iOS по ссылке");
+        } if (os == 1) {
```

Comment:
Тут символ ';' в конце строки лишний.
Из-за него работать не будет. 
Ну и лучше в фигурные скобки вложенное условие вынети:
```suggestion
        if (os == 0) {
            if (years < 2015) {
                System.out.println("Установите облегченную версию приложения для iOS по ссылке");
            } else {
                System.out.println("Установите версию приложения для iOS по ссылке");
            }
        } 
        if (os == 1) {
```